mp/game/momentum/resource/momentum_english_ref.res

```diff
@@ -273,22 +273,24 @@
         // "HUD Settings"
         "MOM_Settings_Tab_HUD" "HUD Settings"
         "MOM_Settings_Speedometer_Label" "Speedometer"
-        "MOM_Settings_Speedometer_Show" "Display absolute speed of player."
-        "MOM_Settings_Speedometer_Horiz_Show" "Display horizontal speed of player."
-        "MOM_Settings_Speedometer_Show_Jump" "Display velocity of player's last jump (XY only)."
-        "MOM_Settings_Speedometer_Show_StageEnter" "Display the absolute UPS speed of the player when entering a stage or zone."
-        "MOM_Settings_Speedometer_Show_StageEnter_TT" "Also displays a stage enter speed comparison."
-        "MOM_Settings_Speedometer_Unit_Labels" "Display unit labels (UPS, KM/H, MPH, Energy)."		
-        "MOM_Settings_Speedometer_Color_Type_Label" "Speedometer Color Mode:"
-        "MOM_Settings_Speedometer_Color_Type_None" "No color"
-        "MOM_Settings_Speedometer_Color_Type_1" "Based on acceleration"
-        "MOM_Settings_Speedometer_Color_Type_2" "Based on velocity compared to max"
-        "MOM_Settings_Speedometer_Color_TT" "Changes colorization of absolute & horizontal speedometers only."
+        "MOM_Settings_Speedometer_GameType_Label" "Gamemode:"
+        "MOM_Settings_Speedometer_Type_Label" "Type:"
+        "MOM_Settings_Speedometer_Type_Absolute" "Absolute Speedometer"
+        "MOM_Settings_Speedometer_Type_Horiz" "Horizontal Speedometer"
+        "MOM_Settings_Speedometer_Type_LastJump" "Last Jump Velocity"
+        "MOM_Settings_Speedometer_Type_StageEnterExit" "Stage Enter/Exit Velocity"
+        "MOM_Settings_Speedometer_Show" "Display this speedometer."
```

Comment:
Should this have a period? I didn't see any pictures of any UI so I don't know the context of this string

Reply:
Fits with the key presses & strafe sync stuff but not the timer.
![image](https://user-images.githubusercontent.com/9014762/83776454-0b3d3d00-a63d-11ea-84d8-a6d0bdbd98f8.png)

Reply:
Ah alright, I'll clean this up in my options panel branch.